test/criaParametrosBusca.test.js

```diff
@@ -0,0 +1,25 @@
+const criaDataFim = require('../src/util/criaDataFim');
+const parametrosParaBusca = require('../src/util/criaParametrosBusca');
+
+describe('teste para a funcao parametrosBusca', () => {
+  test('Busca despesas Atradas ou Pendentes, datas definidas', () => {
+    const exemplo = { situacao: new RegExp(/Atrasada|Pendente/, 'i'), vencimento: { $gte: '2024-01-31', $lte: '2024-06-10' } };
+    const response = parametrosParaBusca({ situacao: 'Atrasada,Pendente', dataInicio: '2024-01-31', dataFim: '2024-06-10' });
```

Comment:
o que acontece se `dataInicio` ou `dataFim` for nulo?

Reply:
Será removido dos parametros de busca. Dentro dessa funcao tem uma outra que é responsavel por validar os valores recebidos.